lib/ide/src/view/text_editor.rs

```diff
@@ -45,54 +88,65 @@ impl TextEditor {
         let text_size    = 16.0;
         let properties   = TextFieldProperties {font,text_size,base_color,size};
         let text_field   = TextField::new(&world,properties);
+        let key_listener = None;
+        let controller_clone = controller.clone_ref();
+        let text_field_clone = text_field.clone();
+        executor::global::spawn(async move {
+            if let Ok(content) = controller_clone.read_content().await {
+                text_field_clone.write(&content);
```

Comment:
I don't think that `TextField`'s `write` is what we need here, it says that it appends text under the cursor. We want to set the whole TextField contents to `content`.